.gitattributes

```diff
@@ -0,0 +1,6 @@
+.clang* export-ignore
+.git* export-ignore
+.jenkins* export-ignore
+.appveyor.yml export-ignore
```

Comment:
```suggestion
appveyor.yml export-ignore
```

Reply:
No I renamed it

Reply:
https://github.com/kokkos/kokkos/blob/fde97cae8467e38c1fc05fff622fdf16993e7bbf/.appveyor.yml